paddle/fluid/framework/ir/mkldnn/matmul_activation_mkldnn_fuse_pass.cc

```diff
@@ -0,0 +1,313 @@
+                                        "relu6",
+                                        "sigmoid",
+                                        "sqrt",
+                                        "swish",
+                                        "tanh"};
+
+  std::vector<std::string> matmul_types = {"matmul"};
+
+  for (const auto& matmul_type : matmul_types)
+    for (auto& act_type : act_types) {
+      std::unordered_map<std::string, std::string> attrs_map;
+
+      if (act_type == "swish")
+        attrs_map.emplace("beta", "fuse_alpha");
+      else if (act_type == "relu6")
+        attrs_map.emplace("threshold", "fuse_alpha");
+      else if (act_type == "hard_sigmoid") {
+        attrs_map.emplace("slope", "fuse_alpha");
+        attrs_map.emplace("offset", "fuse_beta");
+      } else if (act_type == "clip") {
+        attrs_map.emplace("min", "fuse_alpha");
+        attrs_map.emplace("max", "fuse_beta");
+      } else {
+        attrs_map.emplace("alpha", "fuse_alpha");
+        attrs_map.emplace("beta", "fuse_beta");
+      }
+      FuseMatmulAct(graph, matmul_type, act_type, attrs_map);
+    }
+}
+
+void MatmulActivationMkldnnFusePass::FuseMatmulAct(
+    Graph* graph,
+    const std::string& matmul_type,
+    std::string& act_type,
+    const std::unordered_map<std::string, std::string>& attrs_map) const {
+  PADDLE_ENFORCE_NOT_NULL(
+      graph, platform::errors::InvalidArgument("Graph cannot be nullptr."));
+  FusePassBase::Init(matmul_type + "_" + act_type + "_mkldnn_fuse_pass", graph);
+
+  GraphPatternDetector gpd;
+  patterns::OperatorActivation matmul_act_pattern(
+      gpd.mutable_pattern(), "matmul_activation_mkldnn_fuse");
+  matmul_act_pattern(matmul_type, act_type);
+
+  int found_matmul_activation_count = 0;
+  auto handler = [&](const GraphPatternDetector::subgraph_t& subgraph,
+                     Graph* g) {
+    VLOG(4) << "handle " + matmul_type + "+" + act_type + " fuse";
+
+    if (!IsCompat(subgraph, g)) {
+      LOG(WARNING) << "matmul_activation_mkldnn_fuse_pass op compat failed.";
+      return;
+    }
+
+    GET_IR_NODE_FROM_SUBGRAPH(matmul, preceding_op, matmul_act_pattern);
+    GET_IR_NODE_FROM_SUBGRAPH(matmul_out, preceding_op_out, matmul_act_pattern);
+    GET_IR_NODE_FROM_SUBGRAPH(activation, activation, matmul_act_pattern);
+    GET_IR_NODE_FROM_SUBGRAPH(
+        activation_out, activation_out, matmul_act_pattern);
+
+    OpDesc* matmul_op = matmul->Op();
+    OpDesc* act_op = activation->Op();
+
+    for (const auto& attrs : attrs_map) {
+      if (act_op->HasAttr(attrs.first)) {
+        matmul_op->SetAttr(attrs.second, act_op->GetAttr(attrs.first));
+      }
+    }
+
+    if (act_type == "gelu" && activation->Op()->HasAttr("approximate")) {
+      act_type = BOOST_GET_CONST(bool, activation->Op()->GetAttr("approximate"))
+                     ? "gelu_tanh"
+                     : "gelu_erf";
+      matmul_op->SetAttr("fuse_alpha", 0.0f);
+      matmul_op->SetAttr("fuse_beta", 0.0f);
```

Comment:
elt_act_mkldnn_fuse_pass don't need these attributes to be set explicitly so this pass don't need it either IMHO.
```suggestion
```